docs/api/dialog.md

```diff
@@ -154,7 +155,7 @@ dialog.showOpenDialog(mainWindow, {
 })
 ```
 
-### `dialog.showSaveDialog([browserWindow, ]options)`
+### `dialog.showSaveDialogSync([browserWindow, ]options)`
```

Comment:
ah, this just got fixed yesterday actually! You'll see it on rebase

Reply:
Sorry about that. Maybe I hadn't properly installed my fork. Now it's linting on its own and this one passes. Thanks for your patience.

Reply:
thank YOU for the contribution 🚀